core/src/ast/operator.rs

```diff
@@ -18,6 +19,7 @@ impl ToSql for UnaryOperator {
             UnaryOperator::Minus => "-".to_owned(),
             UnaryOperator::Not => "NOT ".to_owned(),
             UnaryOperator::Factorial => "!".to_owned(),
+            &UnaryOperator::BitNot => "~".to_owned(),
```

Comment:
1. `&` is unnecessary. we can remove this.
2. could we use `Bitwise` prefix rather than `Bit` because we already use `bitwise` for other operators. e.g. `BinaryOperator::BitwiseAnd` and `BinaryOperator::BitwiseShiftLeft`

Reply:
1. Ok. It's my mistake.
2. Ok. I will change it to BitwiseNot.